### ISSUE TITLE
Chrome screen reader extension breaks viewer

### ISSUE THREAD

**Issue by JessicaFrank** [CCALI/A2JViewer#153]
Tested a free screen reader extension for Chrome (https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn/related). When I installed it, it would read from the browser as expected and home/landing pages for a2j.org and viewerdev. However, when the viewer launched, I'd get a blank viewer page and a couple errors in the console. Removal of the extension = everything working as expected with the viewer and GIs loading. 

Not urgent, but this should be noted in case users have this installed in their browsers. 

This chrome extension was listed as a top ten free screen reader tool on several lists I looked at. 
![image (7)](https://user-images.githubusercontent.com/7662028/134714502-e86b5125-1866-4713-8c67-cbfacde9dd8a.png)

**Comment by johnmayer** [comment]
This one has many more downloads and much better reviews - try this - https://chrome.google.com/webstore/detail/read-aloud-a-text-to-spee/hdhinadidafjejdhmfkjgnolgimiaplp?hl=en

Google has EOLed their screen reader - they aren't doing any more development on it  - don't waste time trying to get a2j to work with it.
